en/middleware/micrortps_manual_code_generation.md

```diff
@@ -83,15 +64,15 @@ $ python Tools/generate_microRTPS_bridge.py -s msg/sensor_baro.msg -r msg/sensor
 
 Code is generated for the *Client*, *Agent*, *CDR serialization/deserialization* of uORB messages, and the definition of the associated RTPS messages (IDL files). 
 
-Code for the bridge can be found here:
+Code for the bridge by default in the manual generation can be found here:
 
-- *Client*: **build\__BUILDPLATFORM_/src/modules/micrortps_client/**
-- *Agent*: **build\__BUILDPLATFORM_/src/modules/micrortps_bridge/**
+- *Client*: **src/modules/micrortps_bridge/micrortps_client/**
+- *Agent*: **src/modules/micrortps_bridge/micrortps_agent/**
 
 
 ### uORB serialization code
 
-Serialization functions are generated for the [supported uORB topics](#supported-uorb-messages). For example, the following functions would be generated for the *sensor_combined.msg*:
+Serialization functions are generated in the PX4 normal compilation process for all the uORB topics. For example, the following functions would be generated for the *sensor_combined.msg*:
```

Comment:
OK, in that case I might change this as below (essentially clarifying that it happens for all forms of generation). 

> Serialization functions are generated for **all** the uORB topics as part of the normal PX4 compilation process (and also for manual generation).